<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointInfoDlgClass</class>
 <widget class="QDialog" name="PointInfoDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointInfoDlg</string>
  </property>
  <widget class="QLabel" name="xMinLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>xmin</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="xMin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="xMax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="xMaxLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>xmax</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="yMin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yMaxLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>ymax</string>
   </property>
  </widget>
  <widget class="QLabel" name="yMinLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>ymin</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="yMax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="zMax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="zMinLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>zmin</string>
   </property>
  </widget>
  <widget class="QLabel" name="zMaxLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>zmax</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="zMin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total points</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="totalPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="firstPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="firstPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>first return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="secondPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="secondPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>second return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="thirdPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="thirdPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>third return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="fourthPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fourthPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fourth return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="fifthPtN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fifthPtNLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fifth return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ptDensity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ptDensityLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>point density</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>PointInfoDlgClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
